core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceServiceUnitTestsSpec.scala

```diff
@@ -0,0 +1,307 @@
+package org.broadinstitute.dsde.rawls.workspace
+
+import akka.actor.ActorSystem
+import bio.terra.profile.model.{CloudPlatform, ProfileModel}
+import bio.terra.workspace.client.ApiException
+import bio.terra.workspace.model.{CloneWorkspaceResult, JobReport}
+import org.broadinstitute.dsde.rawls.TestExecutionContext
+import org.broadinstitute.dsde.rawls.billing.BillingProfileManagerDAO
+import org.broadinstitute.dsde.rawls.config.MultiCloudWorkspaceConfig
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord.JobType
+import org.broadinstitute.dsde.rawls.dataaccess.{
+  LeonardoDAO,
+  SamDAO,
+  SlickDataSource,
+  WorkspaceManagerResourceMonitorRecordDao
+}
+import org.broadinstitute.dsde.rawls.dataaccess.workspacemanager.WorkspaceManagerDAO
+import org.broadinstitute.dsde.rawls.model.{
+  CreationStatuses,
+  RawlsBillingProject,
+  RawlsBillingProjectName,
+  RawlsRequestContext,
+  RawlsUserEmail,
+  SamWorkspaceActions,
+  UserInfo,
+  Workspace,
+  WorkspaceRequest,
+  WorkspaceState
+}
+import org.joda.time.DateTime
+import org.mockito.ArgumentMatchers
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{doAnswer, doReturn, never, spy, verify, when}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.flatspec.AnyFlatSpecLike
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar
+
+import java.util.UUID
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+// kept separate from MultiCloudWorkspaceServiceSpec to separate true unit tests from tests with awkward actors
+class MultiCloudWorkspaceServiceUnitTestsSpec
+    extends AnyFlatSpecLike
+    with MockitoSugar
+    with Matchers
+    with ScalaFutures {
+  implicit val executionContext: ExecutionContext = TestExecutionContext.testExecutionContext
+  implicit val actorSystem: ActorSystem = ActorSystem("MultiCloudWorkspaceServiceSpec")
+
+  behavior of "cloneMultiCloudWorkspaceAsync"
+
+  it should "pass a request to clone an azure workspace to cloneAzureWorkspaceAsync" in {
+    val sourceWorkspaceName = "source-name"
+    val sourceWorkspaceNamespace = "source-namespace"
+    val sourceWorkspace = Workspace.buildMcWorkspace(
+      sourceWorkspaceNamespace,
+      sourceWorkspaceName,
+      UUID.randomUUID().toString,
+      DateTime.now(),
+      DateTime.now(),
+      "creator",
+      Map(),
+      WorkspaceState.Ready
+    )
+    val destWorkspaceRequest = WorkspaceRequest("dest-namespace", "dest-name", Map())
```

Comment:
Could you include a policy in the request so we make sure it gets passed through?